examples/flutter_infinite_list/lib/posts/bloc/post_bloc.dart

```diff
@@ -77,5 +77,5 @@ class PostBloc extends Bloc<PostEvent, PostState> {
     throw Exception('error fetching posts');
   }
 
-  bool _hasReachedMax(int postsCount) => postsCount < _postLimit ? false : true;
+  bool _hasReachedMax(int postsCount) => postsCount < _postLimit ? true : false;
```

Comment:
The old logic seems correct to me. If the postsCount is less than the limit, we have not reached the max. The updated implementation seems incorrect.

For example, if `postsCount` is 0 and `_postLimit` is 20, with the proposed changes `_hasReachedMax` would evaluate to `true` which is incorrect.